src/showMovie.js

```diff
@@ -0,0 +1,26 @@
+const showMovies = (movieData) => {
+  movieData.forEach((el) => {
+    /* eslint-disable */
+    const { title, poster_path } = el;
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Addressed 👍🏼